pycbc/types/timeseries.py

```diff
@@ -622,7 +622,7 @@ def to_frequencyseries(self, delta_f=None):
 
     @_nocomplex
     def cyclic_time_shift(self, dt):
-        """Shift the data by a given number of seconds
+        """Shift the data and timestamps by a given number of seconds
         
         Shift the data by a given number of seconds. This may
```

Comment:
suggest 'Shift the data and its timestamps around the timeseries vector by a given number of seconds.  To just change the time stamps, do ts.start_time += dt'